app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mobile_navigation"
+    app:startDestination="@+id/nav_home">
+
+    <fragment
```

Comment:
Using the `<fragment` tag is deprecated. Use `FragmentContainerView` instead

Reply:
If I'm using FragmentContainerView the app is crashing some how

Reply:
FragmentContainerView requires a different set up to use with nav controller than the <fragment tag

Reply:
Found the google issue with the right way to use it: https://issuetracker.google.com/issues/142847973#comment4

```
val navHostFragment = supportFragmentManager.findFragmentById(R.id.my_nav_host_fragment) as NavHostFragment
val navController = navHostFragment.navController
```